routes/landslide.py

```diff
@@ -0,0 +1,102 @@
+from flask import render_template, jsonify
+import logging
+
+from . import routes
+from fetch_data import get_landslide_db_row, get_place_data
+from validate_data import place_name_and_type
+
+logger = logging.getLogger(__name__)
+
+
+def validate_community_id(community_id):
```

Comment:
I think to keep this really simple you can just test the supplied strings.  We shouldn't have translation from upper/lowercase unless things are manually entered, which is sort of not a supported use case right?  And it'd be great to untangle this thread of multiple code blocks that do the same WFS request.